odata-csdl/14 Vocabulary and Annotation.md

```diff
@@ -3161,6 +3209,15 @@ the base term or its base term etc. need not be specified again.
 For collection-valued properties the absence of a property value
 expression is equivalent to specifying an empty collection as its value.
 
+If a dynamically provided property value is not acceptable for the type of the property or
+its [facets](#TypeFacets), does not meet the constraints imposed by
+its [nullability](#Nullable) or by
+its annotations from the Validation vocabulary [OData-VocValidation](#ODataVocValidation),
+no value is effectively provided for the property.
+In this case the client MAY apply the rules for absent properties:
```

Comment:
By using MAY, we do not force clients to treat such "errors" in the same way as actually omitted properties.